### PR TITLE
Try to make ingress conformance tests run again

### PR DESCRIPTION
**Pull request by tcnghia**
Test grid https://testgrid.knative.dev/net-istio#continuous&width=20 shows that 44751b604 started the breakage.

### CODE REVIEW

File: vendor/knative.dev/test-infra/scripts/library.sh [hunk 1]
```diff
@@ -389,25 +389,8 @@ function mktemp_with_extension() {
...
-<testsuites>
-	<testsuite tests="1" failures="1" time="0.000" name="$1">
-		<testcase classname="" name="$2" time="0.0">
-			${failure}
-		</testcase>
-	</testsuite>
-</testsuites>
-EOF
+  local xml="$(mktemp_with_extension "${ARTIFACTS}"/junit_XXXXXXXX xml)"
+  run_kntest junit --suite="$1" --name="$2" --err-msg="$3" --dest="${xml}" || return 1
```
Comment (chizhg) [inline vendor/knative.dev/test-infra/scripts/library.sh:393]: This is the `test-infra` revision that is known to have issues..

**Comment by tcnghia** [comment]
/close

ingress test have been missing since after https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/knative_net-istio/93/pull-knative-net-istio-integration-tests/1258036453116481536

**Comment by tcnghia** [comment]
/test pull-knative-net-istio-latest

TestUpdate is a known flake.

**Comment by tcnghia** [comment]
/assign @chizhg 

@chizhg can you take another look to see if this test-infra version has issue?  I think we can follow up with 1.13->1.14 in a later PR.  I'd like to check this in first to ensure we are covered by integration testing.  Thanks.

**Pull request merged**